docs/plugins/subsonicplaylist.rst

```diff
@@ -0,0 +1,33 @@
+Subsonic Playlist Plugin
+========================
+
+The ``subsonicplaylist`` plugin allows to import playlists from a subsonic server.
+This is done by retrieving the track infos from the subsonic server, searching
+them in the beets library and adding the playlist names to the
+`subsonic_playlist` tag of the found items. The content of the tag has the format:
+
+    subsonic_playlist: ";first playlist;second playlist"
+
+To get all items in a playlist use the query `;playlist name;`.
```

Comment:
Wouldn't this require that there be a trailing ; in the tag as well?

Reply:
Your right. The example is wrong as every playlist named has a trailing semicolon